example/symfony31/app/AppKernel.php

```diff
@@ -29,6 +29,19 @@ public function registerBundles()
         return $bundles;
     }
 
+    public function boot()
+    {
+        parent.boot();
```

Comment:
While this is technically valid syntax, it does not do what you think! This takes the value of the constant `parent`, and attempts string concatenation with the value obtained from evaluating the global function `boot`.

Reply:
Huh, thanks @GrahamCampbell - looks like this should've been `parent::boot()`

Reply:
Yeh. GitHub's syntax highlighter had trouble believing it wasn't a method invocation too. :)